apps/database/src/models/ScreenshotDiff.ts

```diff
@@ -109,15 +109,17 @@ export class ScreenshotDiff extends Model {
 
   static sortDiffByStatus = `CASE \
     WHEN "compareScreenshotId" IS NULL \
-      THEN 3 -- removed
+      THEN 30 -- removed
     WHEN "baseScreenshotId" IS NULL \
       AND "compareScreenshot"."name" ~ '${ScreenshotDiff.screenshotFailureRegexp}' \
       THEN 0 -- failure
     WHEN "baseScreenshotId" IS NULL  \
-      THEN 2 -- added
+      THEN 20 -- added
+    WHEN "score" IS NOT NULL AND "score" > 0 and "stabilityScore" < 60 \
+      THEN 11 -- changed and flaky
     WHEN "score" IS NOT NULL AND "score" > 0 \
```

Comment:
```suggestion
    WHEN "score" IS NOT NULL  -- changed
        THEN (CASE
          WHEN "stabilityScore" < 60
            THEN 11 -- flaky
          ELSE 10 -- not flaky
        END) 
    WHEN "score" IS NOT NULL AND "score" > 0 \
```

Reply:
We can do the same optimization for failure / added. it avoid testing condition twice.